src/harness/reference_models/iap/test_data/pal_1.json

```diff
@@ -0,0 +1,24 @@
+{
+  "palId": "pal_1",
+  "licensee": "CBSD Operator No.1",
+  "userID": "user_0",
+  "registrationInformation": {
+    "frn": "test_fcc_frn_pal_0",
+    "callSign": "call_sign_pal_0",
+    "licenseConditions": "license_pal_0"
+  },
+  "license": {
+    "id": "test_fcc_id_pal_0"
+  },
```

Comment:
userId and other registration info are the same as pal_0. Update to match the pal_1 record